zend_abstract_interface/functions/tests/functions.cc

```diff
@@ -111,7 +20,7 @@ static bool zai_test_execute_script(const char *file) {
 }
 
 TEST_CASE("call internal functions with the embed SAPI", "[zai]") {
-    if (!zai_test_embed_init()) REQUIRE(false);
+    if (!zai_sapi_spinup()) REQUIRE(false);
```

Comment:
Maybe write it like `REQUIRE(zai_sapi_spinup());` ?

Reply:
🤯